dotnet_grpc_bench/GreeterServer/Program.cs

```diff
@@ -16,37 +16,45 @@
 
 #endregion
 
-using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Routing;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
+using GreeterServer.Services;
+using GreeterServer;
+using System;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.WebHost.ConfigureKestrel(options =>
+{
+	options.AddServerHeader = false;
+	options.ListenAnyIP(50051, listenOptions =>
+	{
+		listenOptions.Protocols = HttpProtocols.Http2;
+	});
+});
 
-namespace GreeterServer
+builder.WebHost.ConfigureLogging(logging =>
```

Comment:
This can be simplified using the Logging property on the builder. builder.Logging.ClearProviders()